src/configuration/app/storage.md

```diff
@@ -11,8 +11,20 @@ Many applications still require the ability to write and store files, however.
 
 ## Disk
 
+Your _plan storage size_ specifies the maximum total space available to _all_ applications and services.
+
+When deploying your project, the sum of all `disk` keys define din `.platform.app.yaml` and `.platform/services.yaml` 
+must be *equal or less* than the _plan storage size_. For example, if your _plan storage size_ is 5 GB, you can assign:
+
+- 2 GB to your application, 3 GB to your database
+- 1 GB to your application, 4 GB to your database
+- 1 GB to your application, 1 GB to your database, 3 GB to your Elasticsearch service and so on.
```

Comment:
```suggestion
- 1 GB to your application, 1 GB to your database, 3 GB to your Elasticsearch service
- etc.
```

Reply:
Also, use asterisks rather than dashes for the bullets.  That's what we've been doing in most of the docs.